logging/google/cloud/logging/handlers/transports/background_thread.py

```diff
@@ -19,137 +19,224 @@
 
 import atexit
```

Comment:
Definitely want a `from __future__ import print_statement` if you actually want all of these `print`-s.